src/responder/nss/nss_get_object.c

```diff
@@ -171,7 +171,9 @@ hybrid_domain_retry_data(TALLOC_CTX *mem_ctx,
                                           input_name);
     }
 
-    cache_req_data_set_hybrid_lookup(hybrid_data, true);
+    if (hybrid_data != NULL) {
```

Comment:
What if `hybrid_data` is NULL? Should we do anything about it?

This patch will avoid printing the error message and that's fine, but I wonder if we should do anything else.

Reply:
IIUC it is expected to be NULL for any request that is not `CACHE_REQ_GROUP_BY_ID` or `CACHE_REQ_GROUP_BY_NAME`, so the problem is just the log message due to the code path taken prior to this PR.

Reply:
This function - `hybrid_domain_retry_data()` - is only  used in `nss_get_hybrid_object_step()`, where it is expected NULL can be returned ("This request cannot be retried").